auction_keeper/main.py

```diff
@@ -69,6 +70,9 @@ def __init__(self, args: list, **kwargs):
                             help="Do not take opportunities to create new auctions")
         parser.add_argument('--kick-only', dest='bid_on_auctions', action='store_false',
                             help="Do not bid on auctions")
+        parser.add_argument('--deal-for', type=str, nargs="+",
```

Comment:
would be good to have a startup sample in readme file for clarity

Reply:
There's a whole new section in there for it; check line 247.